yaksh/views.py

```diff
@@ -73,13 +74,30 @@ def is_moderator(user):
     if user.groups.filter(name='moderator').exists():
         return True
 
+
 def add_to_group(users):
     """ add users to moderator group """
     group = Group.objects.get(name="moderator")
     for user in users:
         if not is_moderator(user):
             user.groups.add(group)
 
+
+def extract_files(questions_file):
+    if zipfile.is_zipfile(questions_file):
+        zip_file = zipfile.ZipFile(questions_file, 'r')
+        zip_file.extractall()
+
+
+def read_json(json_file, user):
```

Comment:
This is related to Question model then should be in models either as model method or in Question manager.